src/store/modules/assets.js

```diff
@@ -847,6 +849,31 @@ const actions = {
       console.error(err)
       throw err
     }
+  },
+
+  async getPendingAssets({ commit }, daily = false) {
+    const assets = []
+    cache.assets.forEach(asset => {
+      let isPending = false
+      asset.tasks.forEach(taskId => {
+        const task = tasksStore.state.taskMap.get(taskId)
+        if (!isPending) {
+          if (daily) {
+            if (task.last_comment_date) {
+              const lastCommentDate = moment(task.last_comment_date)
+              const yesterday = moment().subtract(1, 'days')
+              isPending =
+                task.task_status_short_name === 'wfa' &&
```

Comment:
Testing an editable task status name seems unreliable.

Why not test on the "is_feedback_request" property of the task status?

Reply:
Thx, it's fixed